src/components/structures/auth/SoftLogout.js

```diff
@@ -48,7 +63,14 @@ export default class SoftLogout extends React.Component {
             domainName,
             userId,
             displayName,
+            loginView: LOGIN_VIEW.LOADING,
+
+            busy: false,
+            password: "",
+            errorText: "",
         };
+
+        this._initLogin();
```

Comment:
call this from `componentDidMount` rather, [as recommended](https://reactjs.org/docs/react-component.html#componentdidmount)?